lnrpc/rpc.proto

```diff
@@ -1830,9 +1854,35 @@ message ChanPointShim {
     bytes pending_chan_id = 5;
 }
 
+message PsbtShim {
+    /**
+    The size of the output to be created as the channel point for this channel
+    funding.
+    */
+    int64 amt = 1;
+
+    /**
+    If non-zero, then this will be used as the pending channel ID on the wire
+    protocol to initiate the funding request. This is an optional field, and
+    should only be set if the responder is already expecting a specific pending
+    channel ID.
+    */
+    bytes pending_chan_id = 2;
```

Comment:
Not sure we need this, but I guess it can't hurt. 

One other thing we could add here, is a serialized PSBT to serve as the base for the transaction itself. This could be used to progressively create a set of combined channel openings across several nodes. This isn't blocking in my mind though.

Reply:
It's used to identify which shim we are referring to. To validate the PSBT we need to know the amount and multisig script that we expect to be in there. We can only get these through the shim which is stored in the reservation.